src/modules/logger/logged_topics.cpp

```diff
@@ -53,6 +53,7 @@ void LoggedTopics::add_default_topics()
 	add_optional_topic("autotune_attitude_control_status", 100);
 	add_optional_topic("camera_capture");
 	add_optional_topic("camera_trigger");
+	add_optional_topic("can_interface_status", 10);
```

Comment:
```suggestion
	add_optional_topic("can_interface_status", 100);
```
Probably should be 10Hz -> 100ms intervals which is the definition of this parameter:
https://github.com/PX4/PX4-Autopilot/blob/710286da722beae72a488af52b6c1e603f68eea7/src/modules/logger/logged_topics.h#L116

Reply:
Sorry, just saw this now. 

This should actually be `add_optional_topic_multi("can_interface_status", 100);`, to get the multiple instances right. My bad.

Can you commit it directly, @dagar, or do you want me to open a new pr?